pinot-common/src/main/java/org/apache/pinot/common/response/broker/BrokerResponseNative.java

```diff
@@ -557,4 +558,16 @@ public void addToExceptions(QueryProcessingException processingException) {
   public int getExceptionsSize() {
     return _processingExceptions.size();
   }
+
+
+  @JsonProperty("requestId")
+  @Override
+  public String getRequestId() {
+    return _requestId;
+  }
+
+  @Override
```

Comment:
Also annotate (JsonProperty) the setter? (Although seems the annotation is redundant)